--- PKGBUILD.orig	2015-05-22 20:12:51.226909285 +0200
+++ PKGBUILD	2015-05-22 20:14:04.683993900 +0200
@@ -35,6 +35,7 @@
     httpd.service
     arch.layout
     lua53_fixes.patch
+    mod_userdir.patch
 )
 sha256sums=('ad6d39edfe4621d8cc9a2791f6f8d6876943a9da41ac8533d77407a2e630eae4'
             'SKIP'
@@ -42,7 +43,8 @@
             '875903831634edf35d8d57e9a51bacb818255ecb3bfff29627f03e43d1ab65c3'
             '14d0a775eea7f8c66ba55399a9ad1e4fd29e0302f479a15b28bbfc754c5aa347'
             'dda05c6e76f12624e418ca18a36f2e90ec1c5b1cc52fed7142fce6076ec413f3'
-            'e2c3fe0747908971449dba8f1526f008fc27b9ff5c3b490ea9d43a2a5fe8c1e4')
+            'e2c3fe0747908971449dba8f1526f008fc27b9ff5c3b490ea9d43a2a5fe8c1e4'
+            'aaf843dd27c19e38d390109551e84834dca49d87bba71210edc719e67a596acf')
 validpgpkeys=('A93D62ECC3C8EA12DB220EC934EA76E6791485A8') # Jim Jagielski
 
 prepare() {
@@ -55,6 +57,7 @@
 
   cat "${srcdir}/arch.layout" >> config.layout
 
+  patch -Np0 -i "${srcdir}/mod_userdir.patch"
   patch -p3 < ../lua53_fixes.patch
 }
 
