--- PKGBUILD.orig	2014-03-11 16:45:13.549652167 +0100
+++ PKGBUILD	2014-03-11 17:34:24.315101824 +0100
@@ -34,13 +34,15 @@
     httpd.logrotate
     httpd.service
     arch.layout
+    mod_userdir.patch
 )
 sha256sums=('64368d8301836815ae237f2b62d909711c896c1bd34573771e0ee5ad808ce71b'
             'SKIP'
             '63da1a420f4714a3e7af2672d28384419cc7eedbe7bf35baebd02938fabc15bf'
             '875903831634edf35d8d57e9a51bacb818255ecb3bfff29627f03e43d1ab65c3'
             '14d0a775eea7f8c66ba55399a9ad1e4fd29e0302f479a15b28bbfc754c5aa347'
-            'dda05c6e76f12624e418ca18a36f2e90ec1c5b1cc52fed7142fce6076ec413f3')
+            'dda05c6e76f12624e418ca18a36f2e90ec1c5b1cc52fed7142fce6076ec413f3'
+            'aaf843dd27c19e38d390109551e84834dca49d87bba71210edc719e67a596acf')
 
 build() {
   cd httpd-${pkgver}
@@ -51,6 +53,7 @@
       -i docs/conf/httpd.conf.in
 
   cat "${srcdir}/arch.layout" >> config.layout
+  patch -Np0 -i "${srcdir}/mod_userdir.patch"
 	
   ./configure --sbindir=/usr/bin \
       --enable-layout=Arch \
