--- PKGBUILD.orig	2016-03-18 20:16:11.195025421 +0100
+++ PKGBUILD	2016-03-18 20:17:14.871972895 +0100
@@ -35,13 +35,15 @@
     httpd.logrotate
     httpd.service
     arch.layout
+    mod_userdir.patch
 )
 sha256sums=('0644b050de41f5c9f67c825285049b144690421acb709b06fe53eddfa8a9fd4c'
             'SKIP'
             '63da1a420f4714a3e7af2672d28384419cc7eedbe7bf35baebd02938fabc15bf'
             '0bbbfae23a917b2fce0bf8f900f60319b50769224a96314e9301a75ccd078e16'
             'f574bac6d5f398e7a649fc0e1ca66ff01ad4ef34dac71258e93d8a9d9a2b3960'
-            'dda05c6e76f12624e418ca18a36f2e90ec1c5b1cc52fed7142fce6076ec413f3')
+            'dda05c6e76f12624e418ca18a36f2e90ec1c5b1cc52fed7142fce6076ec413f3'
+            'aaf843dd27c19e38d390109551e84834dca49d87bba71210edc719e67a596acf')
 validpgpkeys=('A93D62ECC3C8EA12DB220EC934EA76E6791485A8') # Jim Jagielski
 
 prepare() {
@@ -53,6 +55,8 @@
       -i docs/conf/httpd.conf.in
 
   cat "${srcdir}/arch.layout" >> config.layout
+
+  patch -Np0 -i "${srcdir}/mod_userdir.patch"
 }
 
 build() {
